Gemfile

```diff
@@ -5,17 +5,16 @@ ruby File.read('.ruby-version').chomp
 gem 'capybara'
 gem 'rake'
 gem 'rspec', '~> 3.8'
-gem 'webmock', '~> 3.6'
```

Comment:
Are the changes to the Gemfile intentional? I don't think we can drop middleman as a dependency?